homeassistant/components/ecovacs/vacuum.py

```diff
@@ -39,13 +47,16 @@
 ATTR_ERROR = "error"
 ATTR_COMPONENT_PREFIX = "component_"
 
+SERVICE_GET_POSITIONS = "get_positions"
```

Comment:
```suggestion
SERVICE__RAW_GET_POSITIONS = "raw_get_positions"
```

Reply:
Is there a specific reason for the double underscore in const?

Reply:
Nope only a typo :)